docker/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+---
+version: '3'
+services:
+  kdcproxy:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+    container_name: kdcproxy-apache
+    restart: unless-stopped
+    ports:
+      - "443:443"
+    environment:
+      - KDCPROXY_CONFIG=/config/kdcproxy.conf
+      - SERVER_NAME=${SERVER_NAME:-selfsign.rausys.de}
```

Comment:
his looks custom to your deployment or something, I am not sure we should have this file at all in the repository, sounds like configuration each deployment should deal with on their own.